.devcontainer/container/reload_apache2.php

```diff
@@ -1,3 +1,7 @@
 <?php
 
-exec("nohup service apache2 reload > /dev/null 2>&1 &");
+if (file_exists('/reload.bak') && strlen(file_get_contents('/reload.bak')) > 1) {
+    echo "\033[1mReloading Apache...\033[0m\n\n\033[1mPress 'Reload Window' when prompted.\033[0m\n";
+    exec("nohup service apache2 reload > /dev/null 2>&1 &");

```

Comment:
It would be good to add some error handling or logging in case the Apache reload fails. This can help in debugging issues in the future. [medium]